### PR TITLE
(fix) O3-2757: Change all anchor tag cursors from text to pointer type.

### PR DESCRIPTION
**Pull request by Pradipram**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
The links on the empty state are replaced with Carbon buttons (kind = ghost).

### before changes
image

### after changes
image

## Related Issue
https://openmrs.atlassian.net/jira/software/c/projects/O3/issues/O3-2757?filter=allopenissues

### CODE REVIEW

**Comment by sherrif10** [comment]
Kindly share a full screen shot of the screen with in all other PRs, they dont look clear.

**Comment by Pradipram** [comment]
@sherrif10 
#### before changes
https://github.com/openmrs/openmrs-esm-patient-chart/assets/101059602/34ea9afc-b38d-440b-99a9-ca2cd7341453

### after changes
https://github.com/openmrs/openmrs-esm-patient-chart/assets/101059602/cfabeba8-81e5-44b5-b9a4-678eb1f6ed7f

**Review by sherrif10 — APPROVED** [review]
LGTM  .

**Review by subhamkumarr — APPROVED** [review]

File: packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.scss [hunk 1]
```diff
@@ -16,6 +16,10 @@
   margin: 1.3125rem;
 }
 
+:global(.cds--link) {
```
Comment (vasharma05) [inline packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.scss:19]: This override is meant to be in [esm-styleguide overrides](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-styleguide/src/_overrides.scss)

**Review by vasharma05 — DISMISSED** [review]

**Comment by vasharma05** [comment]
Better solution to this is to change that link to ghost button

**Comment by Pradipram** [comment]
@vasharma05 
I have changed the link to the Carbon Button
Please review this.

**Comment by vasharma05** [comment]
Hi @denniskigen @brandones !
Seeking your review on replacing the link to the Button in the empty state.
Thanks!
image

**Review by brandones — APPROVED** [review]
The button implementation appears to conform to [the designs](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/64074bd857a530318edcd817). LGTM.

**Pull request merged**